src/main/java/com/google/gcp_variant_transforms/library/SchemaGeneratorImpl.java

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.FieldList;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.StandardSQLTypeName;
+import com.google.common.collect.ImmutableList;
+import htsjdk.variant.vcf.VCFHeader;
+
+
+
+/**
+ * Service to create BigQuery Schema from VCFHeader 
+ */
+public class SchemaGeneratorImpl implements SchemaGenerator {
+
+  public static class FieldDescriptionConstants {
+    public static final String REFERENCE_NAME = "Reference name.";
+    public static final String START_POSITION = "Start position. Corresponds to the first " +
+                                                "base of the string of reference bases.";
+    public static final String END_POSITION = "End position. Corresponds to the first base " +
+                                              "after the last base in the reference allele.";
+    public static final String REFERENCE_BASES = "Reference bases.";
+    public static final String NAMES = "Variant names (e.g. RefSNP ID).";
+    public static final String QUALITY = "Phred-scaled quality score (-10log10 prob(call " +
+                                         "is wrong)). Higher values imply better quality.";
+    public static final String FILTER = "List of failed filters (if any) or \"PASS\" " +
+                                        "indicating the variant has passed all filters.";
+    public static final String CALLS = "One record for each call.";
+    public static final String CALLS_SAMPLE_ID = "Unique ID (type INT64) assigned to each " +
```

Comment:
@tneymanov we are not hashing the `call_name`, right?
In that case we use [this description](https://github.com/googlegenomics/gcp-variant-transforms/blob/4eb3d7d2e88598c3197b4efbeb14a48acf1ec6f7/gcp_variant_transforms/libs/schema_converter.py#L126).